tests/solidity_test_helpers/MockUniverse.sol

```diff
@@ -0,0 +1,315 @@
+        setExtraDisputeBondRemainingToBePaidOutValue = _setExtraDisputeBondRemainingToBePaidOutValue;
+    }
+    function setIncreaseExtraDisputeBondRemainingToBePaidOut(bool _setIncreaseExtraDisputeBondRemainingToBePaidOutValue) public {
+        setIncreaseExtraDisputeBondRemainingToBePaidOutValue = _setIncreaseExtraDisputeBondRemainingToBePaidOutValue;
+    }
+    function setDecreaseExtraDisputeBondRemainingToBePaidOut(bool _setDecreaseExtraDisputeBondRemainingToBePaidOutValue) public {
+        setDecreaseExtraDisputeBondRemainingToBePaidOutValue = _setDecreaseExtraDisputeBondRemainingToBePaidOutValue;
+    }
+    function setCalculateFloatingValue(uint256 _setCalculateFloatingValueValue) public {
+        setCalculateFloatingValueValue = _setCalculateFloatingValueValue;
+    }
+    function setTargetReporterGasCosts(uint256 _setTargetReporterGasCostsValue) public {
+        setTargetReporterGasCostsValue = _setTargetReporterGasCostsValue;
+    }
+    function setMarketCreationCost(uint256 _setMarketCreationCostValue) public {
+        setMarketCreationCostValue = _setMarketCreationCostValue;
+    }
+    function setIsParentOf(bool _setIsParentOfValue) public {
+        setIsParentOfValue = _setIsParentOfValue;
+    }
+    function setIsContainerForReportingWindow(bool _setIsContainerForReportingWindowValue) public {
+        setIsContainerForReportingWindowValue = _setIsContainerForReportingWindowValue;
+    }
+    function setIsContainerForDisputeBondToken(bool _setIisContainerForDisputeBondTokenValue) public {
+        setIisContainerForDisputeBondTokenValue = _setIisContainerForDisputeBondTokenValue;
+    }
+    function setIsContainerForMarket(bool _setIsContainerForMarketValue) public {
+        setIsContainerForMarketValue = _setIsContainerForMarketValue;
+    }
+    function setIsContainerForStakeToken(bool _setIsContainerForStakeTokenValue) public {
+        setIsContainerForStakeTokenValue = _setIsContainerForStakeTokenValue;
+    }
+    function setIsContainerForShareToken(bool _setIsContainerForShareTokenValue) public {
+        setIsContainerForShareTokenValue = _setIsContainerForShareTokenValue;
+    }
+    function setDecrementOpenInterest(bool _setDecrementOpenInterestValue) public {
+        setDecrementOpenInterestValue = _setDecrementOpenInterestValue;
+    }
+    function setIncrementOpenInterest(bool _setIncrementOpenInterestValue) public {
+        setIncrementOpenInterestValue = _setIncrementOpenInterestValue;
+    }
+    /*
+    * Impl of IUniverse and ITyped
+     */
+    function getTypeName() public view returns (bytes32) {
+        return "Universe";
+    }
+    function initialize(IUniverse _parentUniverse, bytes32 _parentPayoutDistributionHash) external returns (bool) {
+        setParentUniverseValue = _parentUniverse;
+        setParentPayoutDistributionHashValue = _parentPayoutDistributionHash;
```

Comment:
Consider having these be assigned a variable like `providedParentUniverseValue` to distinguish them from values that are set by the person mocking, rather than set by the contract under test calling this method.

Reply:
I'm leaning towards ignore the input values and just have the setters used. thoughts?

Reply:
Also, the variables for these should be public so the tests can read them to assert that the caller provided the expected parameters to the `initialize` call.

Reply:
ok on verify input for initialize method, I also want to add a mech to verify a method was called. I'll need to look up a good pattern in python. I'll add that functionality.

Reply:
That is probably reasonable, since this method shouldn't ever be called during testing.  Perhaps just have this method `assert(false)` on the first line to ensure it isn't called?